include/openthread/backbone_router_ftd.h

```diff
@@ -158,6 +159,54 @@ uint8_t otBackboneRouterGetRegistrationJitter(otInstance *aInstance);
  */
 void otBackboneRouterSetRegistrationJitter(otInstance *aInstance, uint8_t aJitter);
 
+/**
+ * This method gets the local Domain Prefix configuration.
+ *
+ * @param[in]  aInstance A pointer to an OpenThread instance.
+ * @param[out] aConfig   A pointer to the Domain Prefix configuration.
+ *
+ * @retval OT_ERROR_NONE       Successfully got the Domain Prefix configuration.
+ * @retval OT_ERROR_NOT_FOUND  No Domain Prefix was configured.
+ *
+ * @sa otBackboneRouterSetDomainPrefix
+ * @sa otBackboneRouterRemoveDomainPrefix
+ *
+ */
+otError otBackboneRouterGetDomainPrefix(otInstance *aInstance, otBorderRouterConfig *aConfig);
+
+/**
+ * This method removes the local Domain Prefix configuration.
+ *
+ * @param[in]  aInstance A pointer to an OpenThread instance.
+ * @param[in]  aPrefix   A pointer to an IPv6 Domain prefix.
+ *
+ * @retval OT_ERROR_NONE          Successfully removed the Domain Prefix configuration.
+ * @retval OT_ERROR_INVALID_ARGS  @p aPrefix is invalid.
+ * @retval OT_ERROR_NOT_FOUND     No Domain Prefix was configured or @p aPrefix doesn't matches.
+ *
+ * @sa otBackboneRouterGetDomainPrefix
+ * @sa otBackboneRouterSetDomainPrefix
+ */
+otError otBackboneRouterRemoveDomainPrefix(otInstance *aInstance, const otIp6Prefix *aPrefix);
+
+/**
+ * This method sets the local Domain Prefix configuration.
+ *
+ * If backbone functionality is enabled, the domain prefix would be added to local server data.
+ *
+ * @param[in]  aInstance A pointer to an OpenThread instance.
+ * @param[in]  aConfig   A pointer to the Domain Prefix configuration.
+ *
+ * @retval OT_ERROR_NONE          Successfully set the Domain Prefix.
+ * @retval OT_ERROR_INVALID_ARGS  @p aConfig is invalid.
+ * @retval OT_ERROR_NO_BUFS       Insufficient space to add the Domain Prefix entry to local server data.
+ *
+ * @sa otBackboneRouterGetDomainPrefix
+ * @sa otBackboneRouterRemoveDomainPrefix
+ *
+ */
+otError otBackboneRouterSetDomainPrefix(otInstance *aInstance, otBorderRouterConfig *aConfig);
```

Comment:
How about generalize this to `otBackboneRouterAddDomainPrefix()`? The limitation to one domain prefix is currently a "SHOULD".